src/components/App/App.js

```diff
@@ -28,14 +30,29 @@ export default class App extends Component {
       }
     })
   }
+  mapPosters = (movies) => {
```

Comment:
i like how you made this into a function so we don't have to keep typing the same things